src/core/stdc/assert_.d

```diff
@@ -17,6 +17,15 @@
 
 module core.stdc.assert_;
 
+version (OSX)
+    version = Darwin;
+else version (iOS)
+    version = Darwin;
+else version (TVOS)
+    version = Darwin;
+else version (WatchOS)
+    version = Darwin;
```

Comment:
Wrt. this: there's a predefined `darwin` version, but it's deprecated (`use OSX instead`) - time to undo that and redefine its semantics maybe? Or add a new `Darwin` one with these semantics?

Reply:
I would prefer to do that. Walter was against that https://github.com/dlang/dmd/pull/5231#issuecomment-151361493. IIRC he regrets that we have a version identifier for Posix as well.